controllers/backplaneconfig_controller_test.go

```diff
@@ -126,7 +128,9 @@ var _ = Describe("BackplaneConfig controller", func() {
 				ObjectMeta: metav1.ObjectMeta{
 					Name: BackplaneConfigName,
 				},
-				Spec: v1alpha1.MultiClusterEngineSpec{},
+				Spec: v1alpha1.MultiClusterEngineSpec{
+					TargetNamespace: "default",
```

Comment:
To test targeting a different namespace this should be something other than default. This is the same as the BackplaneOperatorNamespace defined above.